lib/version.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+let JuttleAdapters = require('juttle/lib/runtime/adapters');
+let juttleVersion = require('juttle/package.json').version;
+let juttleServiceVersion = require('../package.json').version;
+let juttleJSDPVersion = require('juttle-jsdp/package.json').version;
+
+function version() {
+    return {
+        version: juttleServiceVersion,
+        dependencies: {
+            juttle: juttleVersion,
+            'juttle-jsdp': juttleJSDPVersion
+        },
+        adapters: JuttleAdapters.list().map((info) => {
+            return { name: info.adapter, version: info.version };
+        })
+    };
+}
+
+
```

Comment:
Maybe rename to versionList or getVersions or something else more indicative that this isn't just a constant?